README.md

```diff
@@ -30,21 +30,47 @@ python src/car_racing.py
 
 ### Available arguments:  
 ```bash
---mode # choices=['train', 'test']
---window_length # The length of the filter window. Must be a positive integer
---memory_limit  # In bytes (?????)
+--mode # choices=['train', 'test', 'record'] # Record will make a video of the runs
+--window_length # Length of the experience replay window.
+--memory_limit  # Limit of how many observations, action, rewards and terminal states to store.
 --warmup_steps  # Lower learning rate during the warmup steps.
 --target_model_update  # Controls how often the target network is updated (n'th step). 
 --learning_rate  # Set the learning rate.
 --train_interval  # How many steps before the model re-fits the neural network
---steps  # Number of total training steps
---evaluation_episodes  # ?????
+--steps  # Number of total training steps.
+--evaluation_episodes  # Total number of test episodes.
 --load_weights_from  # Load weights from a previous run.
 ```
 
-For example, the following arguments will run the code with a limit of 10,000 steps, with 500 warmup steps during which the learning rate is lowered, and loading weights from a previous run/model. 
+**Training the Model**  
+For example, the following arguments will train the model with a limit of 10,000 steps, with 500 warmup steps during which the learning rate is lowered, and loading weights from a previous run/model. 
```

Comment:
where did you get the info about lowered learning rate? i think warmup steps are just initial steps where no the network is only seeing observations and not training.

Reply:
I can't find it again, but I did find [this ](https://books.google.com/books?id=lc6PDwAAQBAJ&pg=PA307&lpg=PA307&dq=keras-rl+why+warm-up&source=bl&ots=AqlGV5SB2-&sig=ACfU3U2LyKFPcE8rYeLiccwbsHomuA7cDA&hl=en&sa=X&ved=2ahUKEwiYxpe79JXoAhUoHzQIHSoaDS8Q6AEwCXoECAkQAQ#v=onepage&q=keras-rl%20why%20warm-up&f=false)that seems to support the idea it doesn't learn during the warm-up steps.

Reply:
Changed it to: "Number of warm-up steps before learning occurs."